Update philips_js.markdown (#17158)
home-assistant/home-assistant.github.io: source/_integrations/philips_js.markdown

The `philips_js` platform allows you to control Philips TVs which expose the [jointSPACE](http://jointspace.sourceforge.net/) JSON-API.

> If your TV respond to http://<ip.of.tv>:1925/system then you are good to go. In the response you should also be able to see which version of the API ("api_version":{"Major":6...) you should use in this integration.

Reviewer: ```suggestion
If your TV responds to `http://IP_ADDRESS_OF_TV:1925/system` then this integration can be used. In the response, you should also be able to see the version of the API the TV uses (`"api_version":{"Major":6...`).
```

For older TVs follow instructions on how to activate the API and if your model is supported [here](http://jointspace.sourceforge.net/download.html). Note that not all listed, jointSPACE-enabled devices will have JSON-interface running on port 1925. This is true at least for some models before year 2011.

> Also note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.

Reviewer: ```suggestion
Also, note that version 6 of the API needs to be authenticated by a PIN code displayed on your TV.
```
